honeybadgermpc/batch_reconstruction.py

```diff
@@ -145,17 +247,14 @@ def with_at_most_non_none(data, k):
 
     Reconstruction takes places in chunks of t+1 values
     """
-    fp = field = GF.get(p)
+    fp = GF.get(p)
     poly = polynomials_over(fp)
-    bench_logger = logging.LoggerAdapter(
```

Comment:
This was added to benchmark the time taken to finish the two rounds. We'll need that information when running experiments. Have you moved this somewhere else?

Reply:
Looks like I removed it somehow. I added the benchmark logging back now.